code/game/objects/items.dm

```diff
@@ -645,10 +645,10 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
 	closeToolTip(usr)
 
 /obj/item/MouseDrop_T(obj/item/I, mob/user)
-	if(!user || src == I)
+	if(!user || !iscarbon(user) || src == I)
```

Comment:
```suggestion
	if(!user || user.incapacitated(ignore_lying=TRUE) || src == I)
```
Ghosts are not CONSCIOUS so they are incapacitated.
And i don't think we care about carbonnes of the user, at least not as much as about their ability to manipulate things

(note: am on mobile, please check spellings)